bool linear_dock
bool orientation_dock
bool rack_attach
float64 distance  
float64 goal_x
float64 goal_y
float64 orientation
string rack_no
bool is_rack_detached
---
bool success   # indicate successful run of triggered service
string message # informational, e.g. for error messages
